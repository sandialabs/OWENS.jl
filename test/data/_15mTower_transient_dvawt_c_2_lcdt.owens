_15mTower_transient_dvawt_c_2_lcdt.mesh
_15mTower_transient_dvawt_c_2_lcdt.el
_15mTower_transient_dvawt_c_2_lcdt.ort
_15mTower_transient_dvawt_c_2_lcdt.jnt
[concentrated_nodal_term_file]
_15mTower_transient_dvawt_c_2_lcdt.bc
0 [platform_property_file]
[initial_conditions_file]
1 _15mTower_transient_dvawt_c_2_lcdt.bld [aero_loads_file]
0 [drivetrain_property_file]
[generator_property_file]
0.000000	0.000000	