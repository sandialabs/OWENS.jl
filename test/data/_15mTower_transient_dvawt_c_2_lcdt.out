t	B1N001Fx	B1N001Fy	B1N001Fz	B1N001Mx	B1N001My	B1N001Mz	B1N002Fx	B1N002Fy	B1N002Fz	B1N002Mx	B1N002My	B1N002Mz	B1N003Fx	B1N003Fy	B1N003Fz	B1N003Mx	B1N003My	B1N003Mz	B1N004Fx	B1N004Fy	B1N004Fz	B1N004Mx	B1N004My	B1N004Mz	B1N005Fx	B1N005Fy	B1N005Fz	B1N005Mx	B1N005My	B1N005Mz	B1N006Fx	B1N006Fy	B1N006Fz	B1N006Mx	B1N006My	B1N006Mz	B1N007Fx	B1N007Fy	B1N007Fz	B1N007Mx	B1N007My	B1N007Mz	B1N008Fx	B1N008Fy	B1N008Fz	B1N008Mx	B1N008My	B1N008Mz	B1N009Fx	B1N009Fy	B1N009Fz	B1N009Mx	B1N009My	B1N009Mz	B1N010Fx	B1N010Fy	B1N010Fz	B1N010Mx	B1N010My	B1N010Mz	B1N011Fx	B1N011Fy	B1N011Fz	B1N011Mx	B1N011My	B1N011Mz	B1N012Fx	B1N012Fy	B1N012Fz	B1N012Mx	B1N012My	B1N012Mz	B1N013Fx	B1N013Fy	B1N013Fz	B1N013Mx	B1N013My	B1N013Mz	B1N014Fx	B1N014Fy	B1N014Fz	B1N014Mx	B1N014My	B1N014Mz	B1N015Fx	B1N015Fy	B1N015Fz	B1N015Mx	B1N015My	B1N015Mz	B1N016Fx	B1N016Fy	B1N016Fz	B1N016Mx	B1N016My	B1N016Mz	B1N017Fx	B1N017Fy	B1N017Fz	B1N017Mx	B1N017My	B1N017Mz	B1N018Fx	B1N018Fy	B1N018Fz	B1N018Mx	B1N018My	B1N018Mz	B1N019Fx	B1N019Fy	B1N019Fz	B1N019Mx	B1N019My	B1N019Mz	B1N020Fx	B1N020Fy	B1N020Fz	B1N020Mx	B1N020My	B1N020Mz	B1N021Fx	B1N021Fy	B1N021Fz	B1N021Mx	B1N021My	B1N021Mz	B1N022Fx	B1N022Fy	B1N022Fz	B1N022Mx	B1N022My	B1N022Mz	B1N023Fx	B1N023Fy	B1N023Fz	B1N023Mx	B1N023My	B1N023Mz	B2N001Fx	B2N001Fy	B2N001Fz	B2N001Mx	B2N001My	B2N001Mz	B2N002Fx	B2N002Fy	B2N002Fz	B2N002Mx	B2N002My	B2N002Mz	B2N003Fx	B2N003Fy	B2N003Fz	B2N003Mx	B2N003My	B2N003Mz	B2N004Fx	B2N004Fy	B2N004Fz	B2N004Mx	B2N004My	B2N004Mz	B2N005Fx	B2N005Fy	B2N005Fz	B2N005Mx	B2N005My	B2N005Mz	B2N006Fx	B2N006Fy	B2N006Fz	B2N006Mx	B2N006My	B2N006Mz	B2N007Fx	B2N007Fy	B2N007Fz	B2N007Mx	B2N007My	B2N007Mz	B2N008Fx	B2N008Fy	B2N008Fz	B2N008Mx	B2N008My	B2N008Mz	B2N009Fx	B2N009Fy	B2N009Fz	B2N009Mx	B2N009My	B2N009Mz	B2N010Fx	B2N010Fy	B2N010Fz	B2N010Mx	B2N010My	B2N010Mz	B2N011Fx	B2N011Fy	B2N011Fz	B2N011Mx	B2N011My	B2N011Mz	B2N012Fx	B2N012Fy	B2N012Fz	B2N012Mx	B2N012My	B2N012Mz	B2N013Fx	B2N013Fy	B2N013Fz	B2N013Mx	B2N013My	B2N013Mz	B2N014Fx	B2N014Fy	B2N014Fz	B2N014Mx	B2N014My	B2N014Mz	B2N015Fx	B2N015Fy	B2N015Fz	B2N015Mx	B2N015My	B2N015Mz	B2N016Fx	B2N016Fy	B2N016Fz	B2N016Mx	B2N016My	B2N016Mz	B2N017Fx	B2N017Fy	B2N017Fz	B2N017Mx	B2N017My	B2N017Mz	B2N018Fx	B2N018Fy	B2N018Fz	B2N018Mx	B2N018My	B2N018Mz	B2N019Fx	B2N019Fy	B2N019Fz	B2N019Mx	B2N019My	B2N019Mz	B2N020Fx	B2N020Fy	B2N020Fz	B2N020Mx	B2N020My	B2N020Mz	B2N021Fx	B2N021Fy	B2N021Fz	B2N021Mx	B2N021My	B2N021Mz	B2N022Fx	B2N022Fy	B2N022Fz	B2N022Mx	B2N022My	B2N022Mz	B3N001Fx	B3N001Fy	B3N001Fz	B3N001Mx	B3N001My	B3N001Mz	B3N002Fx	B3N002Fy	B3N002Fz	B3N002Mx	B3N002My	B3N002Mz	B3N003Fx	B3N003Fy	B3N003Fz	B3N003Mx	B3N003My	B3N003Mz	B3N004Fx	B3N004Fy	B3N004Fz	B3N004Mx	B3N004My	B3N004Mz	B3N005Fx	B3N005Fy	B3N005Fz	B3N005Mx	B3N005My	B3N005Mz	B3N006Fx	B3N006Fy	B3N006Fz	B3N006Mx	B3N006My	B3N006Mz	B3N007Fx	B3N007Fy	B3N007Fz	B3N007Mx	B3N007My	B3N007Mz	B3N008Fx	B3N008Fy	B3N008Fz	B3N008Mx	B3N008My	B3N008Mz	B3N009Fx	B3N009Fy	B3N009Fz	B3N009Mx	B3N009My	B3N009Mz	B3N010Fx	B3N010Fy	B3N010Fz	B3N010Mx	B3N010My	B3N010Mz	B3N011Fx	B3N011Fy	B3N011Fz	B3N011Mx	B3N011My	B3N011Mz	B3N012Fx	B3N012Fy	B3N012Fz	B3N012Mx	B3N012My	B3N012Mz	B3N013Fx	B3N013Fy	B3N013Fz	B3N013Mx	B3N013My	B3N013Mz	B3N014Fx	B3N014Fy	B3N014Fz	B3N014Mx	B3N014My	B3N014Mz	B3N015Fx	B3N015Fy	B3N015Fz	B3N015Mx	B3N015My	B3N015Mz	B3N016Fx	B3N016Fy	B3N016Fz	B3N016Mx	B3N016My	B3N016Mz	B3N017Fx	B3N017Fy	B3N017Fz	B3N017Mx	B3N017My	B3N017Mz	B3N018Fx	B3N018Fy	B3N018Fz	B3N018Mx	B3N018My	B3N018Mz	B3N019Fx	B3N019Fy	B3N019Fz	B3N019Mx	B3N019My	B3N019Mz	B3N020Fx	B3N020Fy	B3N020Fz	B3N020Mx	B3N020My	B3N020Mz	B3N021Fx	B3N021Fy	B3N021Fz	B3N021Mx	B3N021My	B3N021Mz	B3N022Fx	B3N022Fy	B3N022Fz	B3N022Mx	B3N022My	B3N022Mz	B4N001Fx	B4N001Fy	B4N001Fz	B4N001Mx	B4N001My	B4N001Mz	B4N002Fx	B4N002Fy	B4N002Fz	B4N002Mx	B4N002My	B4N002Mz	B5N001Fx	B5N001Fy	B5N001Fz	B5N001Mx	B5N001My	B5N001Mz	B5N002Fx	B5N002Fy	B5N002Fz	B5N002Mx	B5N002My	B5N002Mz	B6N001Fx	B6N001Fy	B6N001Fz	B6N001Mx	B6N001My	B6N001Mz	B6N002Fx	B6N002Fy	B6N002Fz	B6N002Mx	B6N002My	B6N002Mz	B7N001Fx	B7N001Fy	B7N001Fz	B7N001Mx	B7N001My	B7N001Mz	B7N002Fx	B7N002Fy	B7N002Fz	B7N002Mx	B7N002My	B7N002Mz
(s)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)	(N)	(N)	(N)	(N-m)	(N-m)	(N-m)
0.0
0.002
0.004
0.006
0.008
0.01
0.012
0.014
0.016
0.018
0.02
0.022
0.024
0.026
0.028
0.03
0.032
0.034
0.036
0.038
0.04
0.042
0.044
0.046
0.048
0.05
0.052
0.054
0.056
0.058
0.06
0.062
0.064
0.066
0.068
0.07
0.072
0.074
0.076
0.078
0.08
0.082
0.084
0.086
0.088
0.09
0.092
0.094
0.096
